cli/cliui/select.go

```diff
@@ -414,15 +428,17 @@ func (m multiSelectModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 
 		case tea.KeyUp:
 			options := m.filteredOptions()
+			maxIndex := len(options)
 			if m.cursor > 0 {
 				m.cursor--
 			} else {
-				m.cursor = len(options) - 1
+				m.cursor = maxIndex
```

Comment:
I think this logic needs to handle when `EnableCustomInput` is false. Currently it lets you go over the last valid option.